base_link : 'base'

#actuated joints in end effector
joints : ['left_gripper_l_finger_joint']

#open position
pregrasp_posture : [0.0095]
pregrasp_time_from_start : 4.0

#close position
grasp_posture : [0.0125]
grasp_time_from_start : 4.0

#desired pose from end effector to grasp - [x,y,z]
grasp_pose_to_eef : [-0.15, 0, 0]

#ee group name
end_effector_name: 'left_hand'

end_effector_parent_link: 'left_wrist'

